net-fs/samba/samba-4.13.3-r1.ebuild

```diff
@@ -1,32 +1,47 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 PYTHON_COMPAT=( python3_{6..9} )
-PYTHON_REQ_USE='threads(+),xml(+)'
+PYTHON_REQ_USE="threads(+),xml(+)"
 inherit python-single-r1 waf-utils multilib-minimal linux-info systemd pam
 
-MY_PV="${PV/_rc/rc}"
-MY_P="${PN}-${MY_PV}"
+DESCRIPTION="Samba Suite Version 4"
+HOMEPAGE="https://samba.org/"
 
+MY_P="${PN}-${PV/_rc/rc}"
 SRC_PATH="stable"
-[[ ${PV} = *_rc* ]] && SRC_PATH="rc"
-
+if [[ ${PV} = *_rc* ]]; then
+	SRC_PATH="rc"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
```

Comment:
You just inverted the logig here. If `${PV}` contains `_rc` we don't want to have any `KEYWORDS`.

Reply:
ah, right.